README.md

```diff
@@ -1,7 +1,11 @@
 # Maple Proxy Factory
 
+[![Foundry][foundry-badge]][foundry]
 [![Forge CI](https://github.com/maple-labs/maple-proxy-factory/actions/workflows/forge.yaml/badge.svg)](https://circleci.com/gh/maple-labs/maple-proxy-factory/tree/main) [![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](https://www.gnu.org/licenses/agpl-3.0)
```

Comment:
Should we update the license as well?

Reply:
Was discussing with @lucas-manuel as this is already a public repo we can keep this AGPL/ public good